src/utils/ar_data_price_estimator.ts

```diff
@@ -75,4 +78,51 @@ export class ARDataPriceEstimator {
 		const predictedPrice = this.predictor.predictedPriceForByteCount(byteCount);
 		return predictedPrice.winstonPrice;
 	}
+
+	/**
+	 * Estimates the price in AR for a given byte count
+	 *
+	 * @remarks Will fetch pricing data for regression modeling if a regression has not yet been run.
+	 * @remarks The ArDrive community fee is not considered in this estimation
+	 */
+	public async getARPriceForByteCount(byteCount: number): Promise<number> {
+		const winstonPrice = await this.getWinstonPriceForByteCount(byteCount);
+
+		return winstonPrice * arPerWinston;
+	}
+
+	/**
+	 * Estimates the number of bytes that can be stored for a given amount of Winston
+	 *
+	 * @throws On invalid winston values and on any issues generating pricing models
+	 *
+	 * @remarks Will fetch pricing data for regression modeling if a regression has not yet been run.
+	 * @remarks The ArDrive community fee is not considered in this estimation
+	 */
+	public async getByteCountForWinston(winston: number): Promise<number> {
+		if (winston < 0 || !Number.isInteger(winston)) {
+			throw new Error('winston value should be a non-negative integer!');
+		}
+
+		// Lazily generate the price predictor
+		if (!this.predictor) {
+			await this.refreshPriceData();
```

Comment:
Missing test coverage here, but i wouldn't try to cover the thrown error.

Reply:
I expanded the tests to check for the three oracle calls, and tried spying on refreshPriceData. Both methods have not expanded the coverage over this call.